### PR TITLE
feat(docs): add installation instructions for arch linux

### LINKED ISSUE #988: Add Arch Linux AUR packages to installation instructions
I have created and will maintain system packages for Arch Linux users via the [Arch User Repository](https://wiki.archlinux.org/index.php/Arch_User_Repository):

* [languageclient-neovim](https://aur.archlinux.org/packages/languageclient-neovim)
* [languageclient-neovim-bin](https://aur.archlinux.org/packages/languageclient-neovim-bin)

I would like to see these referenced in `INSTALL.md`.

### PR DESCRIPTION
**Pull request by sudoforge**
closes #988

### CODE REVIEW

File: INSTALL.md [hunk 1]
```diff
@@ -17,8 +35,6 @@ None.
 
 > Android is not supported using the `install.sh` script.
 
-Choose steps matching your plugin manager.
```
Comment (autozimu) [inline INSTALL.md:20]: Why this getting removed?

**Comment by autozimu** [comment]
Awesome! One small comment.

File: INSTALL.md (hunk 1 above)
Comment (sudoforge) [reply INSTALL.md:20]: I feel that it's a bit redundant with L27, L38, and L49. I'd be happy to revert the change to this line if you feel like it adds contextual value.

**Comment by autozimu** [comment]
Right. I see it now.

No need to change anything. Will merge in shortly.

**Pull request merged**